### ISSUE TITLE
Remove ECB from blowfish

### ISSUE THREAD

**Issue by vanosg** [eggheads/eggdrop#409]
Reference #281 #384

**Comment by Cizzle** [comment]
I don't think it harms to keep the functionality of decrypting in ECB mode for backward compatibilty. Completely disabling it for encryption is probably a must.
